tests/unit/test_osti_single.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think you're missing tests asserting that the output record are valid.

Reply:
I added a test for schema validation
I intend to add more test